perf/benchmark/runner/runner.py

```diff
@@ -378,6 +454,46 @@ def run_perf_test(args):
                        duration=fortio.duration, size=fortio.size)
 
 
+def run_nighthawk(pod, remote_cmd, labels):
+    # Use a local docker instance of Nighthawk to control nighthawk_service running in the pod
+    # and run transforms on the output we get.
+    docker_cmd = "docker run --rm --network=host {docker_image} {remote_cmd}".format(
+        docker_image=NIGHTHAWK_DOCKER_IMAGE, remote_cmd=remote_cmd)
+    print(docker_cmd, flush=True)
+    process = subprocess.Popen(shlex.split(docker_cmd), stdout=subprocess.PIPE)
+    (output, err) = process.communicate()
+    exit_code = process.wait()
+
+    if exit_code == 0:
+        with tempfile.NamedTemporaryFile(dir='/tmp', delete=True) as tmpfile:
+            dest = tmpfile.name
+            with open("%s.json" % dest, 'wb') as f:
+                f.write(output)
+            print("Dumped Nighthawk's json to {dest}".format(dest=dest))
+
+            # Send human readable output to the command line.
+            os.system(
```

Comment:
Why do we us os.system here and subprocess.Popen above? Do we need error handling here?

Reply:
Good catch.. while this line is not critical (as it emits a human readable version of the results to stdout for convenience) the call below should be consider so.. because that produces what we're after. So failing fast is probably a good idea. Would you be ok with punting this to an issue, and them me creating a PR to address it when @carolynhu is done with landing this?

Reply:
sure, I will create a ticket to list those things we can deal with after nighthawk merging